getting-started/templates/Dependencies/MinIO/README.md

```diff
@@ -45,22 +45,21 @@ The same command can be used to apply configuration changes to your release.
 Configuring object storage is part of the standard process of deploying
 SystemLink Enterprise. The following configuration is required to use MinIO.
 
-In `systemlink-values.yaml`:
+In `OnPrem/storage-values.yaml`:
 
-1. Set `s3Host` to an empty string.
-2. Set `s3ServiceName` to `<release>-minio.<namespace>.svc.cluster.local`.
-3. Set `s3Scheme` to `http`.
+1. Set `s3Host` to `<release>-minio.<namespace>.svc.cluster.local`.
```

Comment:
Note: `s3ServiceName` is just a name ("minio" in this case), not a full host name. The given value is correct when put into `s3Host`. I created AB#3180628 tracking further changes to `s3ServiceName`, if we want any.